examples/v1beta1/xgboost-lightgbm.yaml

```diff
@@ -59,11 +54,11 @@ spec:
             template:
               spec:
                 containers:
-                  - name: xgboostjob
+                  - name: xgboost
                     image: docker.io/kubeflowkatib/xgboost-lightgbm:1.0
                     ports:
                       - containerPort: 9991
-                        name: xgboostjob-port
+                        name: xgboost-port
```

Comment:
https://github.com/kubeflow/tf-operator/blob/08c0d0374cada2c18d19ce9fc07bb598e62b6ccf/pkg/apis/xgboost/v1/constants.go#L22

default container port name is still `xgboostjob-port`?

Reply:
You are right, thanks!